### PR TITLE
Add maintainer for keycloak module

### PR DESCRIPTION
**Pull request by ndclt**
##### SUMMARY

Add the author as maintainer of the keycloak module.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
keycloak

### CODE REVIEW

**Review by resmo — APPROVED** [review]
LGTM

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -717,7 +717,7 @@ files:
       - clustering
       - k8s
   $module_utils/identity/keycloak/:
-     maintainers: eikef
+     maintainers: eikef, ndclt
```
Comment (gundalow) [inline .github/BOTMETA.yml]: Needs to be a space-separated list
```suggestion
     maintainers: eikef ndclt
```

**Review by gundalow — CHANGES_REQUESTED** [review]

File: .github/BOTMETA.yml (hunk 1 above)
Comment (ndclt) [reply .github/BOTMETA.yml]: Thanks. Corrected in db4d7e919e6731ead8ee2a085e49f03eb38d840a.

**Comment by gundalow** [comment]
@eikef Are you OK with this?

**Comment by gundalow** [comment]
@eikef Wondered if you could please take a look at this one-line change. Thanks in advance.

**Comment by rrey** [comment]
@eikef has no activity on github since july 2019. :/

**Comment by gundalow** [comment]
@ndclt welcome to the community!

**Comment by ndclt** [comment]
@gundalow: thanks.

**Pull request merged**